### DISCUSSION TITLE
Download Shared Albums too?

### DISCUSSION THREAD

**Discussion by charleslcso** [icloud-photos-downloader/icloud_photos_downloader#523] [General]
This script is very very nice. 

Wondering if it is possible to also download Shared Albums too?

**Comment by AndreyNikiforov** [comment]
Not at the moment. There are ongoing efforts to make it happen (e.g. #486), so I hope we can get support for shared albums soon.

**Reply by gutessteve** [reply]
Is there anything new here yet?

**Reply by Abitera** [reply]
My wife and I are heavily invested in having all our photos in Shared Albums. I hope for #486 to become a realization!

**Reply by montoyamilton963-beep** [reply]
<img width="428" height="926" alt="Sin título" src="https://github.com/user-attachments/assets/fba95ca9-e08c-418a-8604-982aa56fbdb3" />
H

**Comment by exiom-xyz** [comment]
Any news on this? Related discussions in #947 , #1019 and #1082 
`$ icloudpd --list-libraries` only shows `PrimarySync` for me
I have advanced data protection off and I am the owner of the shared albums

I feel like this is an important thing to have so that it can be easily used by the whole family
